app/src/main/java/org/glucosio/android/adapter/A1cEstimateAdapter.java

```diff
@@ -62,17 +59,16 @@ public View getView(int position, View convertView, ViewGroup parent) {
             }
 
             if (glucoseAverage != null) {
-                if ("mg/dL".equals(db.getUser(1).getPreferred_unit())) {
+                if (DomainConstants.MG_D_L.equals(databaseHandler.getUser(1).getPreferred_unit())) {
```

Comment:
This checking happens a lot in the code, could be useful delegate this to someone to return to the right unit, so we could avoid those `if...else`